packages/switch/test/lion-input-switch.test.js

```diff
@@ -0,0 +1,72 @@
+import { expect, fixture, html } from '@open-wc/testing';
+
+import { LionButton } from '@lion/button';
+
+import '../lion-input-switch.js';
+
+describe('lion-input-switch', () => {
+  it('should have default "input" element', async () => {
+    const el = await fixture(html`
+      <lion-input-switch></lion-input-switch>
+    `);
+    expect(el.slots).to.have.a.property('input');
+    expect(el).to.have.a.property('inputElement');
+    expect(el.slots.input()).to.be.an.instanceof(LionButton);
+    expect(el.inputElement).to.be.an.instanceof(LionButton);
+  });
+
+  it('should trickle "disabled" state to child button', async () => {
```

Comment:
trickle sounds like trickle down which sounds like delegate, but it's more like a sync-up right? So it should sync its disabled state to its child button